lib/foreman_tasks/dynflow/configuration.rb

```diff
@@ -6,6 +6,27 @@
 module ForemanTasks
   # Import all Dynflow configuration from Foreman, and add our own for Tasks
   class Dynflow::Configuration < ::Foreman::Dynflow::Configuration
+    def world_config
+      super.tap do |config|
+        config.backup_deleted_plans = backup_settings[:backup_deleted_plans]
+        config.backup_dir           = backup_settings[:backup_dir]
+      end
+    end
+
+    def backup_settings
+      backup_options = {
+        :backup_deleted_plans => true,
+        :backup_dir => default_backup_dir
+      }
+      settings = SETTINGS[:'foreman-tasks'] && SETTINGS[:'foreman-tasks'][:backup]
+      backup_options.merge!(settings) if settings
+      backup_options
+    end
+
+    def default_backup_dir
+      "./backup"
```

Comment:
As this value is mainly applicable in development, what about using `File.join(Rails.root, 'tmp', 'task-backup')` instead to not make mess in the root foreman directory and need to add it to `.gitignore`

Reply:
No need to add to .gitignore because the backups will end up in foreman/tmp, which is already ignored